pkg/app/load_opts_test.go

```diff
@@ -0,0 +1,24 @@
+package app
+
+import (
+	"reflect"
+	"testing"
+)
+
+// TestLoadOptsDeepCopy tests the DeepCopy function for LoadOpts struct.
+func TestLoadOptsDeepCopy(t *testing.T) {
+	lOld := LoadOpts{
+		Selectors:         []string{"test"},
+		RetainValuesFiles: true,
+		CalleePath:        "test",
+		Reverse:           true,
+		Filter:            true,
+	}
+	lNew := lOld.DeepCopy()
+	lNew.CalleePath = "test1"
+
+	if reflect.DeepEqual(lOld, lNew) {
+		t.Errorf("DeepCopy() did not create a new instance")
+	}
```

Comment:
Perhaps we'd better just check for the equality by `require.Equals(t, lOld, lNew)`? (Use the testify/require for that

```suggestion
	require.Equal(t, lOld, lNew)
```